go/vt/sqlparser/tracked_buffer.go

```diff
@@ -25,6 +25,44 @@ import (
 // function that can be given to TrackedBuffer for code generation.
 type NodeFormatter func(buf *TrackedBuffer, node SQLNode)
 
+// DefaultFormatter applies default formatting.
+func DefaultFormatter(buf *TrackedBuffer, node SQLNode) {
+	node.Format(buf)
+}
+
+// Transformer is used to make changes to SQLNode during formatting.
```

Comment:
The comments need to start with the type name `NodeTransformer` and not `Transformer`